umpost/conversion_driver_esm1p5.py

```diff
@@ -70,27 +82,28 @@ def get_esm1p5_fields_file_pattern(run_id: str):
     return fields_file_name_pattern
 
 
-def get_nc_write_path(fields_file_path, nc_write_dir):
+def get_nc_write_path(fields_file_path, nc_write_dir, date=None):
     """
-    Get filepath for writing NetCDF to based on fields file name.
+    Get filepath for writing netCDF to based on fields file name and date.
 
     Parameters
     ----------
     fields_file_path : path to single UM fields file to be converted.
-    nc_write_dir : path to target directory for writing NetCDF files.
+    nc_write_dir : path to target directory for writing netCDF files.
+    date : tuple of form (year, month, day) associated with fields file data.
 
     Returns
     -------
     nc_write_path : path for writing converted fields_file_path file.
     """
     fields_file_path = Path(fields_file_path)
+    fields_file_name = fields_file_path.name
```

Comment:
A minor simplification is using `fields_file_path.name` directly in the `get_nc_filename()` call:
```suggestion
    nc_write_dir = Path(nc_write_dir)
    nc_file_name = get_nc_filename(fields_file_path.name, date)
```

Reply:
Good idea! Added in c040780